src/utils/clientToSVGTranslator.ts

```diff
@@ -7,6 +7,11 @@ export module Utils {
     private _svg: SVGElement;
     private _measureRect: SVGElement;
 
+    /**
+     * Returns the singleton (per document) ClientToSVGTranslator.
```

Comment:
Not quite right, since it's one per `<svg>`. Better might be
```
Returns the ClientToSVGTranslator for the <svg> containing elem.
If one already exists on that <svg>, it will be returned; otherwise, a new one will be created.
```

See `Dispatcher.Mouse`'s documentation.